saleor/graphql/order/mutations/order_note_add.py

```diff
@@ -79,4 +54,29 @@ def perform_mutation(  # type: ignore[override]
             )
             func = get_webhook_handler_by_order_status(order.status, manager)
             cls.call_event(func, order)
-        return OrderAddNote(order=order, event=event)
+        return OrderNoteAdd(order=order, event=event)
+
+
+class OrderAddNoteInput(BaseInputObjectType):
+    message = graphene.String(
+        description="Note message." + DEPRECATED_IN_3X_INPUT,
+        name="message",
+        required=True,
+    )
+
+    class Meta:
+        doc_category = DOC_CATEGORY_ORDERS
+
+
+class OrderAddNote(OrderNoteAdd):
+    class Arguments(OrderNoteAdd.Arguments):
+        input = OrderAddNoteInput(
+            required=True, description="Fields required to create a note for the order."
+        )
+
+    class Meta:
+        description = "Adds note to the order." + DEPRECATED_IN_3X_MUTATION
+        doc_category = DOC_CATEGORY_ORDERS
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = OrderError
```

Comment:
You should create a dedicated `error` type for the mutation.

Reply:
How should it look like? As far as I can see the `OrderError` is used in all mutations in the `order` app. It was also used in the existing `OrderAddNote` mutation.